.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,48 @@
+---
+name: Feature Request
+about: suggesting new or altered functionality
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+<!-- Have you done the following? -->
+<!--   * read the Code of Conduct? By filing an Issue, you are expected to -->  
+<!--     comply with it, including treating everyone with respect: -->
+<!--     https://github.com/hyperledger/besu/blob/main/CODE_OF_CONDUCT.md -->
+<!--   * Reproduced the issue in the latest version of the software -->
+<!--   * Read the debugging docs: https://besu.hyperledger.org/private-networks/how-to -->
+<!--   * Duplicate Issue check:  https://github.com/search?q=+is%3Aissue+repo%3Ahyperledger/Besu -->
+<!-- Note:  Not all sections will apply to all issue types. -->
```

Comment:
this doesn't all need to be duplicated though, right? if they are now separate templates?